src/AaveV2_Eth_ServiceProviders_20231907/AaveV2_Eth_ServiceProviders_20231907.t.sol

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.19;
+
+import {Test} from 'forge-std/Test.sol';
+
+import {AaveGovernanceV2} from 'aave-address-book/AaveGovernanceV2.sol';
+import {AaveV2Ethereum, AaveV2EthereumAssets} from 'aave-address-book/AaveV2Ethereum.sol';
+import {GovHelpers} from 'aave-helpers/GovHelpers.sol';
+import {IERC20} from 'solidity-utils/contracts/oz-common/interfaces/IERC20.sol';
+
+import {AaveV2_Eth_ServiceProviders_20231907} from './AaveV2_Eth_ServiceProviders_20231907.sol';
+
+contract AaveV2_Eth_ServiceProviders_20231907_Test is Test {
+  function setUp() public {
+    vm.createSelectFork(vm.rpcUrl('mainnet'), 17765420);
+  }
+
+  function test_execute() public {
+    AaveV2_Eth_ServiceProviders_20231907 payload = new AaveV2_Eth_ServiceProviders_20231907();
+
+    uint256 balanceBeforeAUSDC = IERC20(AaveV2EthereumAssets.USDC_A_TOKEN).balanceOf(
+      address(AaveV2Ethereum.COLLECTOR)
+    );
+
+    uint256 balanceBeforeUSDC = IERC20(AaveV2EthereumAssets.USDC_UNDERLYING).balanceOf(
+      address(AaveV2Ethereum.COLLECTOR)
+    );
+
+    assertGt(
+      IERC20(AaveV2EthereumAssets.USDC_UNDERLYING).balanceOf(address(AaveV2Ethereum.COLLECTOR)),
+      0
+    );
+
+    uint256 balanceBeforeAUSDT = IERC20(AaveV2EthereumAssets.USDT_A_TOKEN).balanceOf(
+      address(AaveV2Ethereum.COLLECTOR)
+    );
+    uint256 balanceBeforeADAI = IERC20(AaveV2EthereumAssets.DAI_A_TOKEN).balanceOf(
+      address(AaveV2Ethereum.COLLECTOR)
+    );
+
+    GovHelpers.executePayload(vm, address(payload), AaveGovernanceV2.SHORT_EXECUTOR);
+
+    assertEq(
+      IERC20(AaveV2EthereumAssets.USDC_UNDERLYING).balanceOf(address(AaveV2Ethereum.COLLECTOR)),
+      0
+    );
+
+    assertEq(IERC20(AaveV2EthereumAssets.USDC_UNDERLYING).balanceOf(address(payload)), 0);
+
+    assertEq(IERC20(AaveV2EthereumAssets.USDC_A_TOKEN).balanceOf(address(payload)), 0);
+
+    assertApproxEqAbs(
+      IERC20(AaveV2EthereumAssets.USDC_A_TOKEN).balanceOf(address(AaveV2Ethereum.COLLECTOR)),
+      balanceBeforeAUSDC + balanceBeforeUSDC,
+      1e18
```

Comment:
Why 1e18 while others are 1? Also, isn't v2 aUSDC 6 decimals?

Reply:
So I think it's weird doing approxEqAbs, because it's not exactly 1-to-1 when depositing. 
The difference is 180 USDC total, which is not a lot considering it's 290,000 getting deposited.
So that's why I had done `assertGt` previously. I can go either way, definitely 1e18 was a mistake.

As to why others are 1? There's always some rounding error and I'm withdrawing which is different.